### DISCUSSION TITLE
"Experiment failed to load" error when preloading images on COGNITION.RUN

### DISCUSSION THREAD

**Discussion by lily2brain** [jspsych/jsPsych#1689] [Q&A] [answered]
Hi guys. I have a problem inserting my code in cognition.run. When entering the code, the following message "The experiment failed to load." Appears, looking at the console, apparently it seems to be a problem of pre-loading of the images "Failed to load resource: the server responded with a status of 400 ()" ... the experiment runs normally before being inserted into cognition, I don't know what might be happening ... preloaded all 40 images in cognition, all of them are in jpg format ... if anyone knows how to solve, I'm thankful!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @lily2brain, this is a little tricky to figure out without more information. When you say that you preloaded all your images in cognition.run - do you mean that you uploaded the files in the "Stimuli" section on the left of the code editor? 
Maybe you could share your link? Or source code and/or a screenshot?

**Reply by lily2brain** [reply]
hi, yes, I preloaded it, but I managed to discover the error, I removed all the accents and spaces in the nomination of the stimuli, now it runs normally, thanks!

**Reply by becky-gilbert** [reply]
Great, thanks for the update!
